Bottom: 88bb9e41630bbe144771bce184e52c81cf29951c
Top:    e045b931472d6054c0236a8dc2b6936d1e04270b
Author: Parth Shah <parth@linux.ibm.com>
Date:   2020-02-07 14:36:59 +0530

Refresh of sched-fair-paraphrase

---

diff --git a/kernel/sched/fair.c b/kernel/sched/fair.c
index 94ed9289931b..6c0cb349a02f 100644
--- a/kernel/sched/fair.c
+++ b/kernel/sched/fair.c
@@ -5455,9 +5455,9 @@ static unsigned long capacity_of(int cpu)
 
 enum idle_cpu_type {
 	cpu_busy = 0,
-	cpu_preempted_idle
+	cpu_preempted_idle,
 	cpu_non_preempted_idle,
-	cpu_sched_idle,
+	cpu_sched_idle
 }
 
 /*
